kernel/aster-nix/src/vm/vmo/mod.rs

```diff
@@ -301,8 +301,17 @@ impl Vmo_ {
         })
     }
 
-    /// Commit a range of pages in the VMO, returns the pages in this range.
-    pub fn commit(&self, range: Range<usize>, will_write: bool) -> Result<VmFrameVec> {
+    /// Commit a range of pages in the VMO, and perform the operation
+    /// on each page in the range in turn.
+    pub fn commit_and_operate<F>(
+        &self,
+        range: Range<usize>,
+        mut operation: F,
```

Comment:
Function or closures's names should start with verbs. Change "operation" to "operate".